frontend/src/components/CounterfactualStatistics.jsx

```diff
@@ -20,14 +24,21 @@ const CounterfactualStatistics = (props) => {
     >
       <Card.Body>
         <Card.Title>Counterfactual Statistics</Card.Title>
+        <Card.Text>{`Shift in first restrictions: ${
+          props.shiftFirstRestrictions / to_days
+        } days`}</Card.Text>
+        <Card.Text>{`Shift in lockdown: ${
+          props.shiftLockdown / to_days
+        } days`}</Card.Text>
         {!props.totalCasesCounterfactual ? null : (
           <Card.Text>
             {`Total COVID-19 Cases per Million: ${props.totalCasesCounterfactual
               .toFixed(0)
               .toString()} \n `}
           </Card.Text>
         )}
-        {!(props.totalCasesCounterfactual & props.totalCasesReal) ? null : (
+        {!(props.shiftFirstRestrictions !== 0 || props.shiftLockdown !== 0) ||
```

Comment:
I think this should be a separate ternary check with the following contents:
```
<Card.Text>{`Shift first restrictions: ${props.shiftFirstRestrictions}`}</Card.Text>
<Card.Text>{`Shift lockdown: ${props.shiftLockdown}`}</Card.Text>
```

Reply:
Done.